cloud/config.py

```diff
@@ -34,3 +34,7 @@ class Config:
 
     CACHE_TYPE = "simple"  # Flask-Caching related configs
     CACHE_DEFAULT_TIMEOUT = 100
+
+    # JWT
+    SECRET_KEY = 'my-secret'
```

Comment:
Another reminder that this should be changed :sweat_smile:

Reply:
I saw the PR description but I just wanted another reminder that this is really important to be changed :sweat_smile: